packages/taquito-local-forging/src/michelson/codec.ts

```diff
@@ -135,8 +135,8 @@ export const intEncoder: Encoder<IntValue> = ({ int }) => {
     binary.length <= 6
       ? 6
       : (binary.length - 6) % 7
-      ? binary.length + 7 - ((binary.length - 6) % 7)
-      : binary.length;
+        ? binary.length + 7 - ((binary.length - 6) % 7)
```

Comment:
is this from the lint, or is the whitespace unintended?

Reply:
this is from formatting for the aligning of different ternary operators
should it be reverted? i think it looks a little more clear for the different ternary cases